tools/accuracy_checker/accuracy_checker/launcher/tf_launcher.py

```diff
@@ -224,11 +119,6 @@ def predict(self, inputs, metadata=None, **kwargs):
 
         return results
 
-    def inputs_info_for_meta(self, feed_dict=None):
```

Comment:
please return back removed methods (support lstm , input configuration for input feeder and providing inputs for adapter)

Reply:
In this case better to reopen #1881 because it contains all needed code (support lstm , input configuration for input feeder and providing inputs for adapter and custom evaluator)